### DISCUSSION TITLE
Найцікавіші новини місяця: вересень 2024

### DISCUSSION THREAD

**Discussion by malor** [shopokodu/community#72] [Теми для подкасту]
* https://www.bleepingcomputer.com/news/security/kaspersky-deletes-itself-installs-ultraav-antivirus-without-warning/  https://www.reddit.com/r/antivirus/comments/1fonkyt/kaspersky_us_transition_to_ultraav/ (від пана Дениса)
* https://openai.com/index/introducing-canvas/ (від пана Сергія)
* https://www.techspot.com/news/104872-llama-group-releases-winamp-source-code-but-significant.html

**Comment by rkiyanchuk** [comment]
* [Patent troll Sable pays up, dedicates all its patents to the public!](https://blog.cloudflare.com/patent-troll-sable-pays-up/)
* [Real-time Linux is officially part of the kernel after decades of debate | Ars Technica](https://arstechnica.com/gadgets/2024/09/real-time-linux-is-officially-part-of-the-kernel-after-decades-of-debate/)

**Comment by ikalnytskyi** [comment]
- https://survey.stackoverflow.co/2024/
- https://marker.medium.com/why-dont-tech-companies-pay-their-engineers-to-stay-b9c7e4b751e9
- https://www.cio.com/article/3540579/devs-gaining-little-if-anything-from-ai-coding-assistants.html
